static/downloadables/runPrsCLI.sh

```diff
@@ -49,6 +49,8 @@ version="1.5.0"
 #
 #   Added optional param:
 #       -g biological sex prefered for snp selection
+#       -n number of processes
```

Comment:
Should we change the version number for the CLI in case Justin wants to download it again?

Reply:
I think we will wait to change the version number until Liz and I have finished changing the temp files being created.